plugins/cost-insights/src/utils/graphs.ts

```diff
@@ -23,29 +23,30 @@ import {
   lengthyCurrencyFormatter,
 } from './formatters';
 
-export function formatGraphValue(
-  value: number,
-  _index: number,
-  format?: string,
-) {
-  if (format === 'number') {
-    return value.toLocaleString();
-  }
+export const formatGraphValue =
+  (baseCurrency: string) =>
+  (value: number, _index: number, format?: string) => {
+    if (format === 'number') {
+      return value.toLocaleString();
+    }
 
-  if (value < 1) {
-    return lengthyCurrencyFormatter.format(value);
-  }
+    if (value < 1) {
+      return lengthyCurrencyFormatter(baseCurrency).format(value);
+    }
 
-  return currencyFormatter.format(value);
-}
+    return currencyFormatter(baseCurrency).format(value);
+  };
 
 export const overviewGraphTickFormatter = (millis: string | number) =>
   typeof millis === 'number' ? dateFormatter.format(millis) : millis;
 
-export const tooltipItemOf = (payload: Payload<string, string>) => {
+export const tooltipItemOf = (
+  baseCurrency: string,
+  payload: Payload<string, string>,
+) => {
```

Comment:
you went from changing `(param) => {}` to `(currency) => (param) => {}` for the `formatGraphValue` & (kind of) for the `...Formater` functions, but changed `(param) => {}` to `(currency, param) => {}` here. Is there a reason for it? :)

Reply:
You are right, we can keep a consistency in the approach.